README.md

```diff
@@ -1,3 +1,122 @@
-# vscode
+# VS Code
 
-The action runs VSCode in web browser.
+This action runs an instance of [VS Code](https://code.visualstudio.com/) and runs in in the default web browser.
+
+It requires the name of the image on which to run VS Code and references to the following 5 volumes: data, code, config, notebooks, and results. 
+These volumes will be mounted to /project/data, /project/modules, /project/config, /project/notebooks and /project/results respectively.
+
+### Quick example
+
+```
+jobs:
+  vscode:
+    action: gh:neuro-actions/vscode@master
+    args:
+      image: neuromation/base
+      volumes_data_remote: $[[ volumes.data.remote ]]
+      volumes_code_remote: $[[ volumes.code.remote ]]
+      volumes_config_remote: $[[ volumes.config.remote ]]
+      volumes_notebooks_remote: $[[ volumes.notebooks.remote ]]
+      volumes_results_remote: $[[ volumes.results.remote ]]
+```
+
+## Arguments
+
+### `image`
+
+The name of the image on which to run the VS Code instance.
+
+### Example
+
+```
+args:
+	image: neuromation/base
+```
+
+### `job_name`
+
+Predictable subdomain name that will replace the job's ID in the full job URI. `""` by default
+
+### Example
+
+```
+args:
+	job_name: "vscode-job"
+```
+
+### `http_port`
+
+HTTP port to use for Jupyter. `"8080"` by default
```

Comment:
It's not for Jupyter. It's for the VS Code itself.

Reply:
Fixed in the latest commit